File: jdi tests/jdi-uitest-tutorialtests/src/test/java/com/epam/jdi/uitests/testing/career/common/tests/TestsBase.java
package com.epam.jdi.uitests.testing.career.common.tests;

import com.epam.jdi.uitests.testing.career.page_objects.site.EpamSite;
import com.epam.jdi.uitests.web.testng.testRunner.TestNGBase;
import org.testng.annotations.BeforeSuite;

import static com.epam.jdi.uitests.core.settings.JDISettings.*;
import static com.epam.jdi.uitests.testing.career.page_objects.site.EpamSite.homePage;
import static com.epam.jdi.uitests.web.selenium.elements.composite.WebSite.Init;

/**
 * Created by Roman_Iovlev on 7/13/2015.
 */
public abstract class TestsBase extends TestNGBase {

    @BeforeSuite(alwaysRun = true)
    public static void setUp() throws Exception {
        Init(EpamSite.class);
        homePage.open();
        logger.info("Run Tests");
    }
}

File: C#.Net/Tests/TutorialTests/Career/Tests/TestsBase.cs
using Epam.JDI.Core.Settings;
using Epam.JDI.Web.Selenium.Elements.Composite;
using Epam.Tests.TutorialTests.Career.PageObjects;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Epam.Tests.TutorialTests.Career.Tests
{
    [TestClass]
    public class TestsBase
    {

        [ClassInitialize()]
        public static void setUp()
        {
            WebSite.Init(typeof(EpamSite));
            EpamSite.homePage.Open();
            JDISettings.Logger.Info("Run Tests");
        }

    }
}